nodl_python/nodl/_parsing/_parsing.py

```diff
@@ -65,3 +64,26 @@ def parse(path: Union[str, Path, IO]) -> List[Node]:
     element_tree = etree.parse(path)
 
     return _parse_element_tree(element_tree)
+
+
+def parse_multiple(paths: Sequence[Union[str, Path, IO]]) -> List[Node]:
```

Comment:
I can't imagine a caller outside of this library needing this, can you? Should it be non-public? And perhaps moved to index since it doesn't actually do any parsing on its own?

Reply:
This is called during the secure launch system to get a list of all node/executables a package contains

Reply:
So the launch system figures out the paths for all nodl files and then requests that they all be parsed? How does it find those, by calls into the index module functions? Why do that as opposed to requesting the nodl for a given package?

Reply:
Removed from public api in f2b0031